### ISSUE TITLE
minispike: As a developer, I want to build out a minimal data model supporting the reporting app configuration, so I can prove its effectiveness

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-registration#1553]
#### Description:

This card is to try out the data architecture discussed during the whiteboarding sessions.
The goal is to:
- verify effectiveness
- identify pain points
- Create more implementation cards to bring it to a production standard

Work is in the `Whiteboarding_2024_04_12` miro board

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by pbastia** [comment]
@patriciarussellCAS we can hopefully unlock a bunch of work by doing this early this sprint

**Comment by patriciarussellCAS** [comment]
Tagging @joshgamache and @BallardRobinett - can you please review this ticket and provide any feedback. If there are no issues or questions, then we can remove the backlog refinement tag. 
Thanks!!

**Comment by joshgamache** [comment]
> Tagging @joshgamache and @BallardRobinett - can you please review this ticket and provide any feedback. If there are no issues or questions, then we can remove the backlog refinement tag. Thanks!!

Reviewed ✔️. I didn't identify any missing details and think this is clear to execute on!

**Comment by patriciarussellCAS** [comment]
Tagging @dleard - with Pierre gone, do you have any updates on the status of this ticket?

**Comment by dleard** [comment]
Haven't had time to pick this up at all yet, but I synced with Pierre on Friday on this & he left me some notes on what's been done / where to go next. I hope to make some progress on it this week if time allows
